src/finch/tensor.py

```diff
@@ -245,12 +261,12 @@ def __getitem__(self, key):
         if not isinstance(key, tuple):
             key = (key,)
 
-        if None in key:
+        if not self.is_computed():
             # lazy indexing mode
-            key = _process_lazy_indexing(key)
+            key = _process_lazy_indexing(key, self.ndim)
         else:
             # standard indexing mode
-            key = _expand_ellipsis(key, self.shape)
+            key = _expand_ellipsis(key, self.ndim)
```

Comment:
This isn't correct. The signature is `_expand_ellipsis(key: tuple, shape: tuple[int, ...])` so you need to either reverse it or change `_expand_ellipsis` implementation.

Reply:
This should solve CI errors.